src/github.com/getlantern/flashlight/autoupdate/autoupdate.go

```diff
@@ -48,7 +47,7 @@ func Configure(cfg *config.Config) {
 func enableAutoupdate(cfg *config.Config) {
 	var err error
 
-	httpClient, err = util.HTTPClient(cfg.CloudConfigCA, client.Addr)
+	httpClient, err = proxied.ChainedNonPersistent(cfg.CloudConfigCA)
```

Comment:
@myleshorton this is a good example of where we need an actual *http.Client.  It gets passed to autoupdate, which passes it to go-update which actually does some stuff with the transport.  We could rewrite all that code to accept a RoundTripper instead, but IIRC it's pretty common for Go libraries to accept *http.Clients.